index.html

```diff
@@ -138,13 +138,28 @@ <h1>About Myself</h1>
                         <li class="lang-item"><img src="../src/assets/images/Ellipse_css.svg" alt="img">CSS</li>
                     </ul>
                 </li>
-                <li class="lang-items-title underline">Frameworks<span>></span></li>
-                
-                <li class="lang-items-title underline">Skills<span>></span></li>
-                
+                <li class="lang-items-title underline">Frameworks<span>></span></li>                
+                <li class="lang-items-title underline">Skills<span>></span></li>                
             </ul>
         </section>
-        
+        <section class="cards">
+            <img class="contact-bg" src="../src/assets/images/Contact_me_bg.svg" alt="image">
+            <img class="shape-position" src="../src/assets/images/contact_form_shapes_1.svg" alt="img">
+            <div class="contact">
+                <h2 class="contact-header">Contact me</h2>
+                <div class="contact-test">
+                    If you have an application you are interested in developing, a feature that you need built or a
+                    project that needs coding. I’d love to help with it
+                </div>
+                <form action="https://formspree.io/f/xqkjaeyz" method="POST" class="inputs">
+                    <input class="input-field" placeholder="Your name" value="" type="text" id="name" name="user_name" maxlength="30" required>
+                    <input class="input-field" placeholder="Your email" type="email" id="mail" name="user_email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" required>
+                    <textarea class="input-field" rows="6" placeholder="Write something nice" maxlength="500" id="msg" name="user_message" required></textarea>
+                    <button class="contact-btn" type="submit">Get in touch</button>
+                </form>  
+                <div class="footer-img"><img src="./src/assets/images/footer_image.svg" alt="image"></div>              
+            </div>            
+        </section>
```

Comment:
- Please make sure to stick to the Figma design as much as possible, which includes:
   - [ ] Eliminating the space on the sides of your design.
   - [ ] Making sure that the design is fully extended to the edges of the display.
   - [ ] Making sure that all elements are inside the design and not overflowing outside it (like how the button is displayed).
   - [ ] Adjusting the border color of the inputs to match the design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/186172942-09923a37-4a47-4973-a1fb-535c37f4fd74.png) | ![image](https://user-images.githubusercontent.com/62290545/186170035-6ee56552-29a1-47d6-a1d3-56a39bff4b2a.png) |